formver.compiler-plugin/testData/diagnostics/conversion/pure_function.kt

```diff
@@ -0,0 +1,7 @@
+import org.jetbrains.kotlin.formver.plugin.Pure
+import org.jetbrains.kotlin.formver.plugin.DumpExpEmbeddings
+
+fun <!VIPER_TEXT!>emptyFunction<!>() { }
+
+@Pure
+fun <!VIPER_TEXT!>emptyAnnotatedFunction<!>() { }
```

Comment:
If you're going to put placeholders, I would put ones where the behaviours are as you expect them to be.  Now, the translation to Viper looks like it is doing something very strange.  (Or at least leave a comment, but given how easy it is to return 42 here, I would do that.)